TutorTask120_Spring_2025_Real_Time_Bitcoin_Price_Analysis_with_Amazon_SageMaker--Initial project upload

aneeshfelicity opened the pull request:
Initial project upload

tkpratardan commented:
@aneeshfelicity 
There are too many files here. 

- Please remove unnecessary files (lot of copies of tmp.build) 
- make sure you are not changing anything outside your working directory. (.gitmodules is being changed here) Remove tracking of these files by using `git rm --cached`

tkpratardan reviewed (changes requested):
Go through the  [project guidelines](https://github.com/causify-ai/tutorials/tree/master/DATA605#readme)  to understand the deliverables.

aneeshfelicity commented:
Hi @tkpratardan, thank you for the feedback!

-   Removed all unnecessary and duplicate files (e.g., tmp.build copies).
-   Ensured no changes were made outside the working directory.
-   Added the correct and updated project files.

Please let me know if anything else needs adjustment!
